transcriptic/routes.py

```diff
@@ -165,8 +165,8 @@ def preview_protocol_embed(api_root, preview_id):
     return "{api_root}/runs/preview/{preview_id}.embed".format(**locals())
 
 
-def view_data(api_root, data_id):
-    return "{api_root}/datasets/{data_id}.embed".format(**locals())
+def view_data(api_root, org_id, project_id, run_id, data_id):
+    return "{api_root}/{org_id}/{project_id}/runs/{run_id}/datasets/{data_id}.embed".format(**locals())
```

Comment:
why add the org_id, project_id, and run_id here?

Reply:
seems like an unnecessary burden, given `data_id`s should be unique

Reply:
yup, this ended up being not required, which is great